ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -84,6 +84,14 @@ void CleanupAttributes()
             CleanupAttributes(typeof(SecurityCriticalAttribute).FullName, x => x.ConstructorArguments.Count == 0);
             CleanupAttributes(typeof(AllowPartiallyTrustedCallersAttribute).FullName, x => x.ConstructorArguments.Count == 0);
             CleanupAttributes(typeof(SecurityRulesAttribute).FullName, x => x.ConstructorArguments.Count == 0);
+            RemoveInternalsVisibleToAttribute();
+            RemoveAttributes<AssemblyDelaySignAttribute>(_ => true);
+            RemoveAttributes<AssemblyKeyFileAttribute>(_ => true);
+            RemoveAttributes<AssemblyKeyNameAttribute>(_ => true);
+        }
```

Comment:
Shall I split CleanupAttributes to, say, 3 methods: CleanupAttributes, RemoveInternalVisibleAttributes, and RemoveSigningAttributes? Or move all RemoveAttributes calls to another RemoveAttributes method?